actors/paych/src/lib.rs

```diff
@@ -65,28 +74,6 @@ impl Actor {
         Ok(())
     }
 
-    /// Resolves an address to a canonical ID address and requires it to address an account actor.
-    fn resolve_account(rt: &mut impl Runtime, raw: &Address) -> Result<Address, ActorError> {
-        let resolved = resolve_to_actor_id(rt, raw)?;
-
-        let code_cid = rt
-            .get_actor_code_cid(&resolved)
-            .ok_or_else(|| actor_error!(illegal_argument, "no code for address {}", resolved))?;
```

Comment:
I'm not confident in changing this behaviour of requiring the actor to exist. I think get_actor_code_cid is a perfectly reasonable way to ascertain this. I'm open to debating this, but I think it's a distraction from the main change you're going for here of removing the requirement for the parties to be built-in Account actors.

Improvements here would probably go further to support f4 addresses, perhaps actually initialise them if they don't exist. But let's do it later (or not at all and wait for third parties to improve).

Reply:
Good, I agree. I do dislike using `get_actor_code_cid` as the way to ascertain this, but I will open an issue for that.